### ISSUE TITLE
Display posts on /blog based on when `blog` tag was added

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#9924]
## Idea
This idea is about the blog. https://publiclab.org/blog

It would be nice if posts could be displayed in chronological order **based on when the blog tag was added**.

**Additional context**
Currently, the blog displays posts in order of the date the note was originally posted.
The problem which that sometime creates for people curating the blog is that content that was posted earlier can be tagged blog but it will never show up on the top of publiclab.org/blog.

**Comment by jywarren** [comment]
This is a nice idea! Here are the database queries from the new and old blogs:

https://github.com/publiclab/plots2/blob/02315eb4a298949c8cd9ece772121b645787ddfa/app/controllers/tag_controller.rb#L253-L267

Both would have to be modified, including joining the node_tag table with the same tag id as the initial query, then sorting based on that. I can help if someone wants to take this one up -- maybe @TildaDares or @Manasa2850 might be interested?

**Comment by jywarren** [comment]
For reference, the new blog is at https://publiclab.org/blog2 and it's not 100% complete so hasn't yet replaced the old blog.

**Comment by ebarry** [comment]
Thanks all, would love to see some movement on this issue.

**Comment by ebarry** [comment]
Just tested on stable.publiclab.org, and it **does not seem to be ordering by date that the `blog` tag was added**. What i did to test: 

- went to https://stable.publiclab.org/blog and observed that the current top post is "Remembering Tonawanda" published on 07-01-2020. 
- went to went to https://stable.publiclab.org/dashboard and scrolled back to find an older note -- https://stable.publiclab.org/notes/sashae/06-26-2020/diy-satellite-ground-station with a posting date 06-26-2020 that pre-dates the Tonawanda note. 
- Added `blog` tag on the "DIY Satellite Ground Station" note
- went to https://stable.publiclab.org/blog, and noticed that the "DIY Satellite Ground Station" note that I just tagged `blog` was not shown above the "Remembering Tonawanda" post.
